src/discordcr/client.cr

```diff
@@ -167,22 +180,16 @@ module Discord
     end
 
     private def identify
-      packet = {
-        op: 2,
-        d:  {
-          token:      @token,
-          properties: {
-            :"$os"               => "Crystal",
-            :"$browser"          => "discordcr",
-            :"$device"           => "discordcr",
-            :"$referrer"         => "",
-            :"$referring_domain" => "",
-          },
-          compress:        false,
-          large_threshold: 100,
-        },
-      }.to_json
-      @websocket.send(packet)
+      shard_key = @shard
+      shard_key_array = nil
+
+      if shard_key
```

Comment:
```cr
if shard = @shard
  shard_tuple = shard.values
end
```

No need for an explicit `shard_tuple = nil`, it's nillable outside the if automatically.